res/d3d8.def

```diff
@@ -1,2 +1,5 @@
 EXPORTS
+	ValidatePixelShader
```

Comment:
I don't know if it is necessary but how about adding ordinal value here?

Reply:
Ordinals shouldn't be necessary as far as I'm aware (d3d8to9 has worked fine without any so far, and technically Direct3DCreate8 is @ 5 in the native dll, not wherever MSVC has decided to put it until now :), but I can add them if @crosire wants me to.

Reply:
IIRC I saw the exports at different ordinals in different Windows versions, indicating that the ordinals are not hardcoded for D3D8 (but may be mixing D3D8 with a different DLL right now), so adding them here would bring no benefit.